org.freedesktop.Sdk.Extension.rust-stable.json

```diff
@@ -123,6 +123,25 @@
                 "test -n \"$COMPAT_TARGET\" && cd \"rust-$COMPAT_TARGET\" && ./install.sh --prefix=/usr/lib/sdk/rust-stable --disable-ldconfig --verbose --components=rust-std-$COMPAT_TARGET,rust-analysis-$COMPAT_TARGET"
             ]
         },
+        {
+            "name": "rust-src",
```

Comment:
weird this isn't provided by the binary distribution

Reply:
True...